src/frontends/tensorflow/tests/compilation.cpp

```diff
@@ -72,12 +72,15 @@ TEST_F(CompileModelsTests, ModelWithSplitConvConcat)
 TEST_F(CompileModelsTests, ModelWithShapeOf) {
     auto model = convert_model("shapeof_slice_abs/shapeof_slice_abs.pbtxt");
     ov::Core core;
+    core.set_property("CPU", ov::hint::inference_precision(ov::element::f32));
```

Comment:
should we have a fix on plugin level?

Reply:
CPU plugin on ARM uses f16 inference precision by default for non-convolution networks.
@dmitry-gorokhov do you see a way how to avoid modifying common tests?